docs/metric.md

```diff
@@ -0,0 +1,81 @@
+# Metric
+
+The metric defines the evaluation result for a model.
+
+## Overview
+
+The ProteinGym benchmark uses the [scripts/metric.py](../scripts/metric.py) script to calculate performance metrics for model predictions. The script computes any custom metrics like Spearman correlation.
+
+## How to add a custom metric
+
+Adding a new custom metric is straightforward and requires the following steps:
+
+### Step 1: Add your metric with a `metric_[your_metric_name]` function
+
+Open [scripts/metric.py](../scripts/metric.py) and take the `metric_spearman()` function as an example.
+
+To add a new metric, define a function following this pattern:
+```python
+def metric_<your_metric_name>(actual_values: list[float], predicted_values: list[float]) -> tuple[str, float]:
+    result = custom_calculation(actual_values, predicted_values)
+    return ("Metric Display Name", result)
```

Comment:
I would prefer to just return the result here as that is a more-logical API for a metric function.